package.json

```diff
@@ -12,6 +12,9 @@
       "stage-0"
     ]
   },
+  "react-svg-icons": {
+    "assetPath": "src/img/global/svg-icons"
```

Comment:
I ended up submitted a plea for help as an issue and one of the package
contributors set me straight. I wish I hadn't struggled with it as long
either, but I appreciate your effort helping me to troubleshoot and happy
to have it working. This will probably be a good default for pulling in
most SVG icons for the time being.

On Mon, Jul 11, 2016 at 9:54 AM, Dale John McGrew <notifications@github.com>
wrote:

> In package.json
> <https://github.com/wevote/WebApp/pull/337#discussion_r70294947>:
>
> > @@ -12,6 +12,9 @@
> >        "stage-0"
> >      ]
> >    },
> > +  "react-svg-icons": {
> > +    "assetPath": "src/img/global/svg-icons"
>
> @ixley <https://github.com/ixley> I am glad to hear it, and am sorry you
> had to wrestle with it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/wevote/WebApp/pull/337/files/f10043a9307091971edadfda35be8f46cc85a086#r70294947>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJzlvg9W0VKZwV4al7BOtx2nxxNUIR_ks5qUnVQgaJpZM4JJkCH>
> .
>